src/Microsoft.AspNetCore.Mvc.Razor.Extensions/Resources.resx

```diff
@@ -135,4 +135,7 @@
   <data name="MvcRazorParser_InvalidPropertyType" xml:space="preserve">
     <value>Invalid tag helper property '{0}.{1}'. Dictionary values must not be of type '{2}'.</value>
   </data>
+  <data name="PageDirectiveCannotBeImported" xml:space="preserve">
+    <value>Page directive cannot be imported.</value>
```

Comment:
I was hoping we could include the {ImportsFileName}. Usually the error has the file the error originated in.